### PR TITLE
Add Notation interactive tutorial to website

### LINKED ISSUE #272: Add Notation interactive tutorial to website
@kokamkarsahil helped us to add Notation interactive tutorial to Killercoda: https://killercoda.com/notaryproject/scenario/notation. It's super helpful for newbies to learn and get started with Notation CLI in a browser within just a few minutes. We could add this interactive tutorial to the Notary website:

1. Add the tutorial entry on the landing page. We can replace this button with this entry and name it `Try it` or `Demo`.
image

2. Add the interactive tutorial link in https://notaryproject.dev/docs/quickstart/

### PR DESCRIPTION
**Pull request by kokamkarsahil**
### This PR does the following changes

1. 7cff57998a952b0fa3b2d49d951a78b54f065c46 replaces GitHub button with link to notation scenario and adds SVG icon indicating external link and opens that link in new tab a4687f94c23bedd00927fa64b065a5d3744e394d.

image

2. ded32bf6820170b7d908e893707914ea4d7627c5 adds info about online tutorial on `/docs/quickstart`

image

---

### Preview changes: 

- https://deploy-preview-273--notarydev.netlify.app/
- https://deploy-preview-273--notarydev.netlify.app/docs/quickstart/
Closes #272

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -34,8 +34,14 @@ <h4 class="title">
           Get started
         </button>
       </a>
-      <a href="https://github.com/notaryproject/notation">
-        <button class="button is-medium is-dark demo">GitHub</button>
+      <a href="https://killercoda.com/notaryproject/scenario/notation">
+        <button class="button is-medium is-dark demo">Try it
```
Comment (FeynmanZhou) [inline content/en/_index.html:38]: @kokamkarsahil  Could you set this button to open a new window since it's an external link?

**Review by FeynmanZhou — COMMENTED** [review]
PTAL

**Comment by FeynmanZhou** [comment]
cc @priteshbandi @iamsamirzon @toddysm @zr-msft @yizha1

**Review by toddysm — APPROVED** [review]

**Review by priteshbandi — APPROVED** [review]
LGTM

**Review by zr-msft — APPROVED** [review]
LGTM

**Pull request merged**